ERROR 2021-01-06 19:44:04,457 root : Unable to conect to the server: HTTPConnectionPool(host='mysite', port=8000): Max retries exceeded with url: /json (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f20292bcc88>: Failed to establish a new connection: [Errno -2] Name or service not known'))
ERROR 2021-01-06 19:45:04,519 root : Unable to conect to the server: HTTPConnectionPool(host='mysite', port=8000): Max retries exceeded with url: /json (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f20292bcda0>: Failed to establish a new connection: [Errno -2] Name or service not known'))
ERROR 2021-01-06 19:46:04,585 root : Unable to conect to the server: HTTPConnectionPool(host='mysite', port=8000): Max retries exceeded with url: /json (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f20292bcc50>: Failed to establish a new connection: [Errno -2] Name or service not known'))
